### PR TITLE
Nb/feat/new metrics

### PR DESCRIPTION
**Pull request by Nadav-Barak**
resolves https://github.com/deepchecks/deepchecks/issues/1728

### CODE REVIEW

**Review by JKL98ISR — APPROVED** [review]
just improve docs for the 3 get_something_rate functions

**Review by noamzbr — APPROVED** [review]
Approved with minor comments

File: deepchecks/tabular/metric_utils/additional_metrics.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    """Receive a metric which calculates false positive rate.
+
+    The rate is calculated as: False Positives / (False Positives + True Negatives)
+    Parameters
+    ----------
+    y_true : array-like of shape (n_samples,)
+        True labels.
+    y_pred : array-like of shape (n_samples,)
+        Predicted labels.
+    averaging_method : str, optional (default='per_class')
```
[the remark this answers is not in the document]
Comment (Nadav-Barak) [reply deepchecks/tabular/metric_utils/additional_metrics.py:50]: Would point to our metric guide what completed :)

File: deepchecks/tabular/_shared_docs.py [hunk 2]
```diff
@@ -25,11 +25,6 @@
     timeout in second for the permutation features importance calculation
 scorers : Optional[Mapping[str, Union[str, Callable]]] , default: None
     dict of scorers names to scorer sklearn_name/function
```
[the remark this answers is not in the document]
Comment (Nadav-Barak) [reply deepchecks/tabular/_shared_docs.py:27]: Explanation will need to improve and refer to our metric guide

**Pull request merged**